src/main/java/com/systemmeltdown/robotlib/subsystems/SkidSteerDriveSubBase.java

```diff
@@ -0,0 +1,32 @@
+package com.systemmeltdown.robotlib.subsystems;
+
+import com.systemmeltdown.robotlib.commands.DriveProportional;
+import com.systemmeltdown.robotlib.commands.DriveWithEncoders;
+
+public abstract class SkidSteerDriveSubBase extends SubsystemBase {
+
+    @Override
+    public void initDefaultCommand() {
+        if (this.isFailsafeActive()) {
+            setDefaultCommand(new DriveProportional());
+        } else {
+            //setDefualtCommand(new DriveWithEncoders());
+        }
+    }
+
+    abstract void setLeftSpeed(double speed);
```

Comment:
Let's set an access level for these other than the implicit "package" access level.
I'd suggest protected if this is used in a subclass.